robot_log_visualizer/plotter/color_palette.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2022 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# Released under the terms of the BSD 3-Clause License
+
+import matplotlib.pyplot as plt
+
+
+class ColorPalette:
+    def __init__(self):
+        # Define the color taking from the default matplotlib color palette
+        self.colors = [
+            color["color"] for color in list(plt.rcParams["axes.prop_cycle"])
```

Comment:
Is this documented somewhere?

Reply:
`prop_cycle` provides the color cycle used as `rcParams`. In the future, if one wants to change the default color palette in Matplotlib, they can modify the set  `rcParams` directly and our color_palette will always take the one set See here: https://matplotlib.org/stable/users/explain/customizing.html.

For details on prop_cycle, visit: https://matplotlib.org/stable/users/explain/artists/color_cycle.html


I can add this to a comment in the code, what do you think?

Reply:
> I can add this to a comment in the code, what do you think?

Yes, I think it could be helpful!

Reply:
Done in https://github.com/ami-iit/robot-log-visualizer/pull/81/commits/d4bf5bf96d7ddccde8d24f939c81a16fc12570a0